src/connector/connector-client.ts

```diff
@@ -24,13 +25,14 @@ import { errorIdentity } from 'utils/error-identity'
 import { sendMessageOverDataChannelAndWaitForConfirmation } from './webrtc/helpers/send-message-over-data-channel-and-wait-for-confirmation'
 import { ConnectorClientSubjects } from './subjects'
 import { MessageErrorReasons, Secrets } from './_types'
+import { RadixConnectConfiguration } from 'options/options'
 
 export type ConnectorClient = ReturnType<typeof ConnectorClient>
 
 export const ConnectorClient = (input: {
   target: MessageSources
   source: MessageSources
-  signalingServerBaseUrl: string
+  radixConnectConfiguration: RadixConnectConfiguration
```

Comment:
`radixConnectConfiguration` should be an object containing all values needed for connecting. This would eliminate importing additional config values downstream in webRTC and signaling client.